test/openlr.cc

```diff
@@ -186,14 +187,15 @@ TEST(OpenLR, CreateLinearReference) {
       bearing %= 360;
       lrps.emplace_back(p.lng(), p.lat(), bearing, frc, fow, lrps.empty() ? nullptr : &lrps.back());
     }
+    distances.emplace_back(lrps.back().distance);
```

Comment:
should this be inside each of the `if`'s above and push `distance` or `0` as opposed to `lrps.back().distance`. the LocationReferencePoint constructor is binning those actual distances. i know the poff and noff arent used in this test, but technically this would be wrong, right?

Reply:
scratch that sorry they are used in this test, i forgot and didnt read it 10 lines down :man_facepalming:

Reply:
After discussing offline, we decided to switch to storing the offset percentage as-is, instead of trying to derive the actual length.